test/unit/lib/pa11y.test.js

```diff
@@ -122,7 +122,7 @@ describe('lib/pa11y', () => {
 
 		it('loads the HTML CodeSniffer JavaScript', () => {
```

Comment:
It might be useful to include a test that checks the file is included correctly when `pa11y` is installed as a dependency, rather than used directly.
I'll have to have a think about how that could be done.

Reply:
Could that happen in a separate PR?

Reply:
I'd say that we are actually ok here. As all we would be testing is functionality of `require.resolve` which we should assume works correctly 😏